Bio/motifs/__init__.py

```diff
@@ -292,7 +290,8 @@ def __set_mask(self, mask):
         elif mask is None:
             self.__mask = (1,) * self.length
         elif len(mask) != self.length:
-            raise ValueError("The length (%d) of the mask is inconsistent with the length (%d) of the motif", (len(mask), self.length))
+            raise ValueError("The length (%d) of the mask is inconsistent with the length (%d) of the motif",
+                             (len(mask), self.length))
```

Comment:
Can you revert this unrelated change (at some point we will be apply black style to this file anyway)